chpl/chpl-service/pom.xml

```diff
@@ -411,7 +411,7 @@
                         <source>src/main/java/gov/healthit/chpl/domain/CertificationResult.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertificationResultAdditionalSoftware.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertificationResultTestData.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/CertificationResultTestFunctionality.java</source>
+                        <source>src/main/java/gov/healthit/chpl/functionalityTested/CertificationResultFunctionalityTested.java</source>
```

Comment:
Is there something about these sources that they're supposed to be alphabetized?

Reply:
I don't know, but moved it.